content/blog/2022-05-05-newsletter.md

```diff
@@ -0,0 +1,190 @@
+
+By the way, xmpp.org features [a new page about XMPP RFCs](https://xmpp.org/rfcs/).
+
+### Proposed
+
+The XEP development process starts by writing up an idea and submitting it to the XMPP Editor. Within two weeks, the Council decides whether to accept this proposal as an Experimental XEP.
+
+-   [Pubsub Public Subscriptions](https://xmpp.org/extensions/inbox/pubsub-public-subscriptions.html)
+    -   This specification provides a way to make subscriptions to a node public
+-   [Ephemeral Messages](https://xmpp.org/extensions/inbox/ephemeral-messages-v2.html)
+    -   This specification encourages a shift in privacy settings wrt. logging policies.
+
+### New
+
+-   No new XEPs this month.
+
+### Deferred
+
+If an experimental XEP is not updated for more than twelve months, it will be moved off Experimental to Deferred. If there is another update, it will put the XEP back onto Experimental.
+
+-   No XEPs deferred this month.
+
+### Updated
+
+-   Version 0.4 of [XEP-0356](https://xmpp.org/extensions/xep-0356.html) (Privileged Entity)
+    -   Add "iq" privilege (necessary to implement XEPs such as ).
+    -   Roster pushes are now transmitted to privileged entity with "roster" permission of "get" or "both". This can be disabled.
+    -   Reformulate to specify than only initial stanza and "unavailable" stanzas are transmitted with "presence" pemission.
+    -   Namespace bump. (jp)
+
+### Last Call
+
+Last calls are issued once everyone seems satisfied with the current XEP status. After the Council decides whether the XEP seems ready, the XMPP Editor issues a Last Call for comments. The feedback gathered during the Last Call help improving the XEP before returning it to the Council for advancement to Stable.
+
+-   No Last Call this month.
+
+### Stable (formerly known as Draft)
+
+Info: The XSF has decided to rename 'Draft' to 'Stable'. [Read more about it here](https://github.com/xsf/xeps/pull/1100).
+
+-   No XEPs advanced to Stable this month.
+
+### Deprecated
+
+-   No XEP deprecated this month.
+
+### Call for Experience
+
+A Call For Experience - like a Last Call, is an explicit call for comments, but in this case it's mostly directed at people who've implemented, and ideally deployed, the specification. The Council then votes to move it to Final.
+
+-   No Call for Experience this month.
+
+## Spread the news!
+
+Please share the news on other networks:
```

Comment:
Maybe add the Lemmy here too?

Reply:
Well, as you know that was my critic: Now its effectively official without any discussion and review.

And that way, I am not amused if this is the procedure to get things established...

9d9683c

Reply:
None of the other places are official either, these are just social places where you can spread the news. There's nothing official about it.

Reply:
Yes, that was before I started in the team. I was simply asking to do things more aligned. But its in there now and I think we discussed already.